docs/reference/webhooks/process.md

```diff
@@ -46,6 +64,12 @@ return response;
 
 Receives an object containing:
 
+### context
+
+`unknown`
```

Comment:
Would it be better to type this as `any` instead of `unknown` so folks aren't forced to cast it?

Reply:
Unknown was used due to internal lint rule's prevents any from being used, but I'll change it to any.